fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/MetricUnionAvailabilitySpec.groovy

```diff
@@ -60,17 +62,22 @@ class MetricUnionAvailabilitySpec extends Specification {
         physicalTables = [physicalTable1, physicalTable2] as Set
 
         apiFilters = new ApiFilters()
+
+        availabilitiesToMetricNames = new HashMap<>()
```

Comment:
I see all tests are having

```groovy
availabilitiesToMetricNames.put(availability1, [metric1] as Set)
availabilitiesToMetricNames.put(availability2, [metric2] as Set)
```

We should fill the map in setup then

Reply:
But we fill the map with different set of values based on the test